microservices/realtime/pkg/services/http.go

```diff
@@ -26,21 +29,25 @@ type CriteriaResponse struct {
 	Criteria string `json:"criteria"`
 }
 
-func GetActiveCallsFilter(token string, role string) (*string, error) {
+func GetActiveCallsFilter(token string, role string, filters *utils.Register) (*string, error) {
 	var api string
+	var queryFilters string = ""
 
 	switch role {
 	case ROLE_SUPER_ADMIN:
 		api = config.GetHttpApiPlatform()
+		queryFilters = getFiltersFromRegister(filters)
 	case ROLE_BRAND_ADMIN:
 		api = config.GetHttpApiBrand()
+		queryFilters = getFiltersFromRegister(filters)
 	case ROLE_COMPANY_ADMIN:
 		api = config.GetHttpApiClient()
 	}
 
 	apiUrl := fmt.Sprintf(
-		"%s/my/active_calls/realtime_filter",
+		"%s/my/active_calls/realtime_filter?%s",
 		api,
+		queryFilters,
```

Comment:
For company admin, queryFilters is empty and this calls to `/my/active_calls/realtime_filter?` (ending with a ?), is this a problem? or the `?` without query parameters works as expected?

Reply:
:+1: